scripts/workers/code_upload_worker_utils/make_submission.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+submission_url="$EVALAI_API_SERVER/api/jobs/challenge/$CHALLENGE_PK/challenge_phase/$PHASE_PK/submission/$SUBMISSION_PK"
+submission_curl_base_request="curl --location --request PATCH '$submission_url' --header 'Authorization: Bearer $AUTH_TOKEN'"
```

Comment:
@savish28 have you verified these curl requests locally once?

Reply:
@Ram81 yes.